integration-test/src/test/java/com/klaytn/caver/model/executor/TransactionExecutor.java

```diff
@@ -140,6 +140,13 @@ public BigInteger getNonce() throws IOException {
                 .send().getValue();
     }
 
+    public void fillGasPrice(Transaction.InnerTransaction tx) throws IOException {
+        if (tx.getGasPrice() == null) {
+            String baseFee = caver.rpc.klay.getHeader(DefaultBlockParameterName.LATEST).send().getResult().getBaseFeePerGas();
```

Comment:
Need to take care of the case if we don't have "baseFee" in the header.

Reply:
Please check https://github.com/klaytn/caver-java/pull/341/commits/5a7ebcc23b5adf9ffb94a612730fb1017fc2eb89 Thanks!